src/shared/constants/wysiwyg2.ts

```diff
@@ -26,6 +28,8 @@ export const WYSIWYG2_OPTIONS_AUTHOR = [
 	'link',
 	'separator',
 	'remove-styles',
+	'separator',
+	'code',
```

Comment:
code is een code block invoegen, niet de html code bekijken, dus dit is wss niet wt je wil
de code bekijken is momenteel geen optie van de braft editor

fix everywhere